code/modules/projectiles/guns/medbeam.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/weapon/gun/medbeam
 	name = "Medical Beamgun"
 	desc = "Delivers medical nanites in a focused beam."
+	origin_tech = "combat=6;materials=7;powerstorage=8;programming=7;biotech6"//The ultimate within medical design
```

Comment:
Personally, I don't think these are good numbers; the super ultra high-tier items generally have lower origin tech than what is required to research them.

These will end up being valuable to scientists not just for "I get to heal everything and invalidate medbay!" but also because they'll be able to hit materials 8 and programming 8---meaning even cheaper R&D items and even faster mech production.